### PR TITLE
#60686 Improve the descriptive text that's used for the site icon

### PR DESCRIPTION
**Pull request by johnbillion**
Trac ticket: https://core.trac.wordpress.org/ticket/60686

### CODE REVIEW

File: src/wp-admin/options-general.php [hunk 1]
```diff
@@ -199,7 +199,7 @@ class="<?php echo esc_attr( $classes_for_button ); ?>"
 	<p class="description">
 		<?php
 			/* translators: %s: Site Icon size in pixels. */
-			printf( __( 'Site Icons are what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. They should be square and at least %s pixels.' ), '<code>512 &times; 512</code>' );
+			printf( __( 'The site icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %s pixels.' ), '<code>512 &times; 512</code>' );
```
Comment (aaronjorbin) [inline src/wp-admin/options-general.php:202]: The feature name has always been capitalized and is also capitalized in the GB implementation as of last week
 
```suggestion
			printf( __( 'The Site Icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %s pixels.' ), '<code>512 &times; 512</code>' );
```

File: src/wp-includes/class-wp-customize-manager.php [hunk 2]
```diff
@@ -5199,7 +5199,7 @@ public function register_controls() {
 					'label'       => __( 'Site Icon' ),
 					'description' => sprintf(
 						/* translators: %s: Site Icon size in pixels. */
-						'<p>' . __( 'Site Icons are what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. They should be square and at least %s pixels.' ) . '</p>',
+						'<p>' . __( 'The site icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %s pixels.' ) . '</p>',
```
Comment (aaronjorbin) [inline src/wp-includes/class-wp-customize-manager.php:5202]:
```suggestion
						'<p>' . __( 'The Site Icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %s pixels.' ) . '</p>',
```

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @johnbillion for the PR! Just needs to address @aaronjorbin's feedback.

**Comment by johnbillion** [comment]
Thanks all! https://core.trac.wordpress.org/changeset/57766

**Pull request closed without merging**